src/bfb_to_raw_img.sh

```diff
@@ -94,12 +101,7 @@ fi
 
 #check that file exist and it is a bfb file
 if [ -f "$bfb" ]; then
-    if [[ $bfb == *ubuntu* ]]; then
```

Comment:
maybe we should add a check to verify that bfb file name includes ubuntu or centos